### ISSUE TITLE
Date.setTime() demo doesn't work

### ISSUE THREAD

**Issue by cyrusae** [mdn/interactive-examples#2086]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/setTime

### What specific section or headline is this issue about?

"Try it" console

### What information was incorrect, unhelpful, or incomplete?

The output for me is 
```
> Object {  }
> Object {  }
```

### What did you expect to see?

According to the page, the intended output is
```
console.log(event1);
Thu Jul 01 1999 00:00:00 GMT+0200 (CEST)
Thu Jul 01 1999 00:00:00 GMT+0200 (CEST)
```

### Do you have anything more you want to share?

I'm in PST, if the timezone matters for this.

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/javascript/reference/global_objects/date/settime`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/setTime
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/javascript/reference/global_objects/date/settime/index.md
* Last commit: https://github.com/mdn/content/commit/cf607d68522cd35ee7670782d3ee3a361eaef2e4
* Document last modified: 2021-07-20T18:47:57.000Z

</details>

**Comment by sideshowbarker** [comment]
The fix for this needs to be made in the `live-examples/js-examples/date/date-settime.js` file.

**Comment by NiedziolkaMichal** [comment]
@sideshowbarker That would be possible workaround, but when I checked related issue, I found out that there are few more interactive examples with the same problem. I think it's better to make a global solution, so we don't need to add "toString()" everywhere.

**Comment by sideshowbarker** [comment]
> I think it's better to make a global solution, so we don't need to add "toString()" everywhere.

OK, yeah, you’re right. So we need somebody to work on https://github.com/mdn/bob/pull/745

**Comment by sideshowbarker** [comment]
> OK, yeah, you’re right. So we need somebody to work on [mdn/bob#745](https://github.com/mdn/bob/pull/745)

Correction: I see now that #745 _is_ the fix — it just needs to be reviewed and merged. @wbamberg @mdn/core-yari-dev

**Comment by caugner** [comment]
The fix for this issue (#745) is now live. 🎉
